src/tslint-rules.json

```diff
@@ -108,11 +108,11 @@
 			"call-signature",
```

Comment:
should this be disabled as well?

I think it's unnecessary to explicitly specify the return type, since it can be inferred automatically; if desired, a more specific/generic return type can be specified